src/panel/category_panel.js

```diff
@@ -172,49 +137,4 @@ export default class CategoryPanel {
   removeCategoryMarkers() {
     fire('remove_category_markers', this.pois);
   }
-
-  selectPoi = poi => {
-    const previousMarker = document.querySelector('.mapboxgl-marker.active');
-    if (previousMarker) {
-      previousMarker.classList.remove('active');
-    }
-    if (poi.meta && poi.meta.source) {
-      Telemetry.add('open', 'poi', poi.meta.source,
-        Telemetry.buildInteractionData({
-          id: poi.id,
-          source: poi.meta.source,
-          template: 'multiple',
-          zone: 'list',
-          element: 'item',
-          category: this.categoryName,
-        })
-      );
-    }
-    window.app.navigateTo(`/place/${poi.toUrl()}`, {
-      poi: poi.serialize(),
-      isFromCategory: true,
-      sourceCategory: this.categoryName,
-      layout: layouts.LIST,
-      centerMap: true,
-    });
-    this.highlightPoiMarker(poi);
-  }
```

Comment:
Click on markers is broken on this branch. A listener still depends on this function (see line 30).

Reply:
@amatissart Sorry, it looks I went a bit fast when moving functions downstream and forgot to test.
I put marker-related functions back in the `category_panel.js`.
And to finish the migration in a future PR, I think I'll delegate these tasks to the `scene_category.js` file so the panel itself can be fully unmounted without map markers management being broken.